python/xorbits/_mars/dataframe/datasource/tests/test_datasource_execution.py

```diff
@@ -1209,14 +1209,14 @@ def test_read_parquet_arrow(setup, engine):
             pd.testing.assert_frame_equal(df, r.sort_values("a").reset_index(drop=True))
 
             # test `use_arrow_dtype=True`
-            mdf = md.read_parquet(
-                f"{tempdir}/*.parquet", engine=engine, use_arrow_dtype=True
-            )
-            result = mdf.execute().fetch()
-            assert isinstance(mdf.dtypes.iloc[1], md.ArrowStringDtype)
-            assert isinstance(result.dtypes.iloc[1], md.ArrowStringDtype)
-
             if engine != "fastparquet":
```

Comment:
For `fast_parquet`, `use_arrow_dtype` cannot work?

Reply:
I don't want to not support it, because it need to convert to arrow dtype after reading and has to copy data.

Reply:
Maybe need a check when engine == 'fastparquet" and use_arrow_dtype is True.